go.sum

```diff
@@ -246,6 +246,8 @@ go.mongodb.org/atlas v0.8.1-0.20210617105011-6061b0945dd4 h1:vLw72G0G30fDpTP6rsz
 go.mongodb.org/atlas v0.8.1-0.20210617105011-6061b0945dd4/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
 go.mongodb.org/ops-manager v0.22.1-0.20210617143723-fa7bb89af282 h1:iQjUqcG8DO/541s46U9FpxhE0bGTMypjaUy1bAn7vM0=
 go.mongodb.org/ops-manager v0.22.1-0.20210617143723-fa7bb89af282/go.mod h1:13raq36D7hiSiDbnBGB+MChiRF3HKd9wZPkZHkZnjR8=
+go.mongodb.org/ops-manager v0.22.1-0.20210618124154-28a24c0e3fb0 h1:bT5Mvz0zNJOrsj+G6e9X9V50Wjcy8afj+rges6gm9UQ=
```

Comment:
for some reason it did not delete the previous two lines, can you run `go mod tidy`?

Reply:
TIL

Reply:
Good spot! Done